### ISSUE TITLE
PreMid does not work on Opera or Opera GX

### ISSUE THREAD

**Issue by trvpmilk** [PreMiD/PreMiD#684]
**My PreMid does not work on Opera platforms**
I downloaded an extension to transfer Chrome extensions to Opera GX and I set up the extension by adding my presences from the Presence Store. I put on my game activity on Discord and enabled my **all** presences. None of my presences are visible.

**Expected behavior**
I expected for my presences to be visible.

 - OS: [Windows 10]
 - Browser [Opera GX]
 - Version [My PREMID is up to date and so it Opera GX]

**Comment by RedMephistopheles** [comment]
Bump. I've just installed both the desktop application, as well as the browser addon plus associated "Install Chrome Extensions" addon. Using Windows 10 Home Edition, on Opera GX, version LVL 2 (core: 73.0.3856.415), nearly none of the PreMid presences work. I've found that Reddit, Google, Instagram, Crunchyroll, and SoundCloud seem to work fine. The rest are often hit-or-miss, or nonfunctional all together.
